src/VisualStudio/Core/Def/ExternalAccess/Pythia/PythiaGlobalOptions.cs

```diff
@@ -34,9 +34,9 @@ public bool RemoveRecommendationLimit
         }
 
         private static readonly Option2<bool> s_showDebugInfoOption = new(
-            "InternalFeatureOnOffOptions_ShowDebugInfo", defaultValue: false);
+            "dotnet_show_debug_info", defaultValue: false);
```

Comment:
>dotnet_show_debug_info [](http://example.com/codeflow?start=13&length=22)

I think this should include "intellicode" in the name.

`dotnet_show_intellicode_debug_info`,
`dotnet_remove_intellicode_recommendation_limit`